atd-etl/data_model/README.md

```diff
@@ -0,0 +1,94 @@
+# CRIS Import ETL
+
+This ETL manages the processing and importing of TxDOT CRIS data into the Vision Zero database.
+
+All data processing is managed by a single script, `cris_import.py` which processes both CSV files and CR3 PDF crash reports. The script supports a number of CLI arguments to handle a number of data processing scenarios.
+
+## Quick start
+
+Follow these steps to run the ETL locally. All interactions with AWS S3 occur with against a single bucket which has subdirectores for the `dev` and `prod` environments. If you set your `ENV` to `dev` you can safely run this ETL's S3 operations.
```

Comment:
"All interactions with AWS S3 occur with against a single" 
either remove with or against

Reply:
actually what about removing this part, since its repeated below under environment and then for here just having it say "Set your ENV to dev to safely run this ETL locally"